replacy/__init__.py

```diff
@@ -55,7 +57,9 @@ def __init__(
         forms_lookup=None,
         custom_match_hooks: Optional[ModuleType] = None,
         allow_multiple_whitespaces=False,
-        lemmatizer="pyInflect",
+        lemmatizer="pyInflect",                
+        use_lm=True,
+        testing=False
```

Comment:
These 2 parameters are so we can build the LM stuff,
```python
self.scorer = KenLMScorer(nlp=self.nlp, model=get_default_lm(testing=testing)) if use_lm else None
```

Now that I see that, it seems like directly passing in a `scorer` is cleaner ("Dependency Injection")

Reply:
Yes! Bonus: using lm is optional now.